src/ngraph/frontend/onnx_import/utils/pooling_factory.hpp

```diff
@@ -74,6 +74,17 @@ namespace ngraph
                 ngraph::op::PadType m_auto_pad;
             };
 
+            ///
+            /// \brief      Factory class which generates sub-graphs for ONNX 'local' pooling
```

Comment:
Please explain the meaning of `local` in this context.

Reply:
done, described in `\note` part of doc